src/idp/mod.rs

```diff
@@ -0,0 +1,108 @@
+pub mod error;
+use self::error::Error;
+
+pub mod sp_extractor;
+pub mod verified_request;
+pub mod response_builder;
+
+#[cfg(test)]
+mod tests;
+
+use std::str::FromStr;
+use openssl::{rsa::Rsa, x509, pkey, asn1::Asn1Time};
+use openssl::nid::Nid;
+use openssl::pkey::Private;
+use openssl::bn::{BigNum, MsbOption};
+
+use crate::crypto::{self};
+
+use crate::schema::Response;
+use crate::idp::response_builder::{build_response_template};
+use response_builder::ResponseAttribute;
```

Comment:
```suggestion
use response_builder::{build_response_template, ResponseAttribute};
```
Clean up this import